obspy/io/sac/tests/test_core.py

```diff
@@ -915,6 +917,19 @@ def test_always_sac_reftime(self):
         self.assertAlmostEqual(tr1.stats.sac.a, a, places=5)
         self.assertEqual(tr1.stats.sac.b, b)
 
+    def test_encode_flag(self):
+        """
+        Read SAC file using ASCII encoding by default. User can specify
+        encoding through encoding flag.
+        """
```

Comment:
Please catch all the warnings here - depending on what you use to execute the tests you might not see them but they will clobber the test output.